### PR TITLE
[#7]FEAT(Create Class): create class and class details page

### PR DESCRIPTION
**Pull request by irvinalfaro11**

### CODE REVIEW

**Comment by irvinalfaro11** [comment]
format adjusted  and removed unnecessary comments and files

File: src/app/class/add-class/add-class.component.html [hunk 1]
```diff
@@ -4,13 +4,70 @@
...
+                        <mat-option value="2017-2018">2017-2018</mat-option>
+                        <mat-option value="2018-2019">2018-2019</mat-option>
+                        <mat-option value="2019-2020">2019-2020</mat-option>
+                    </mat-select>
+                  </mat-form-field>
+                </div>
+                <div class="col-md-4">
+                  <mat-form-field style="width:200px !important;" ngClass="form-group label-floating">
+                    <mat-select placeholder="Term"  formControlName="term">
+                        <mat-option value="1">Term 1</mat-option>
```
Comment (johnhigginsavila) [inline src/app/class/add-class/add-class.component.html]: use *ngFor

File: src/app/class/add-class/add-class.component.html [hunk 2]
```diff
@@ -4,13 +4,70 @@
...
+                        <mat-option value="2">Term 2</mat-option>
+                        <mat-option value="3">Term 3</mat-option>
+                    </mat-select>
+                  </mat-form-field>
+                </div>
+                <div class="col-md-4">
+                  <mat-form-field style="width:200px !important;" ngClass="form-group label-floating">
+                    <mat-select placeholder="Cycle"  formControlName="cycle">
```
Comment (johnhigginsavila) [inline src/app/class/add-class/add-class.component.html:29]: use *ngFor

**Review by johnhigginsavila — CHANGES_REQUESTED** [review]
- User *ngFor for select tags

**Review by johnhigginsavila — APPROVED** [review]
-LGTM

**Pull request merged**